src/stories/containers/cookies-policy/cookies-policy-banner.mvvm.ts

```diff
@@ -48,25 +49,58 @@ export const useCookiesPolicyBannerMvvm = ({ cookiesObject }: Props) => {
 
   const setFunctionalTracking = useCallback(
     (val: boolean) => {
-      setCookie('themeTracking', val);
-      setCookie('timestampTracking', val);
+      setCookie('themeTracking', val, {
+        expires: daysToExpire(),
+        path: '/',
+      });
+      setCookie('timestampTracking', val, {
+        expires: daysToExpire(),
+        path: '/',
+      });
     },
     [setCookie]
   );
   const setAnalyticsTracking = useCallback(
     (val: boolean) => {
-      setCookie('analyticsTracking', val);
+      setCookie('analyticsTracking', val, {
+        expires: daysToExpire(),
+        path: '/',
+      });
     },
     [setCookie]
   );
 
+  const setThemeModeCookie = useCallback(() => {
+    const newThemeMode = cookies.themeModeCookie ? cookies.themeModeCookie : 'light';
+    setCookie('themeModeCookie', newThemeMode, {
+      expires: daysToExpire(),
+      path: '/',
+    });
+  }, [cookies.themeModeCookie, setCookie]);
+
   const deletedFunctionalTracking = useCallback(() => {
-    removeCookie('themeTracking');
-    removeCookie('timestampTracking');
+    removeCookie('themeTracking', {
+      expires: daysToDeleteCookie(),
+      path: '/',
+    });
+    removeCookie('timestampTracking', {
+      expires: daysToDeleteCookie(),
+      path: '/',
+    });
   }, [removeCookie]);
 
   const deletedAnalyticsTracking = useCallback(() => {
-    removeCookie('analyticsTracking');
+    removeCookie('analyticsTracking', {
+      expires: daysToDeleteCookie(),
+      path: '/',
+    });
+  }, [removeCookie]);
+
+  const deletedThemeCookie = useCallback(() => {
+    removeCookie('themeModeCookie', {
+      expires: daysToDeleteCookie(),
+      path: '/',
```

Comment:
I found these ways to remove the cookies a bit hard to catch as the date to be deleted is 2 years (ago). Did you try to remove the `expires`? Like this: https://stackoverflow.com/a/69792708/11429218

Note that this is done on the client side, so removing the cookies with the standard library should also work.

If removing the `expires` (but keeping the `path` doesn't work, maybe we should try to use the `max-age` property: look here and example (in SSR) but the principle is the same: https://dev.vamsirao.com/clearing-cookie-in-nextjs

This way is more readable and easy to understand